tests/workspace_bridge/test_security.py

```
@@ -111,10 +211,52 @@ def test_anonymous_cannot_read_host_stats(raw_client: TestClient):
+    ticket_r = raw_client.post("/api/auth/ws-ticket", headers=integrator_headers)
+    assert ticket_r.status_code == 200
+    ticket = ticket_r.json()["ticket"]
+    assert ticket
+    with raw_client.websocket_connect(f"/ws/dashboard?ticket={ticket}") as ws:
+        payload = ws.receive_json()
+        assert "stack" in payload
+        assert "faults" in payload
+
+
+def test_ws_rejected_without_ticket(raw_client: TestClient):
+    with pytest.raises(Exception):
```

Ruff (0.15.15)

[warning] Do not assert blind exception: `Exception`

(B017)